index.html

```diff
@@ -0,0 +1,36 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Document</title>
+    <link rel="stylesheet" href="style.css">
+    <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/4.7.0/css/font-awesome.min.css">
+
+</head>
+<body>
+  <header>
+    <nav class="container">
+      <img id="logo" src="img/image_geometry_1.png" alt="">
+      <div id="bars">
+        <button><i class="fa fa-bars" style="color:black;"></i></button>
+      </div>
+    </nav>
+  </header>
+  <section class="heading">
+    <h1 id="name">
+      Kashif Saleem
+    </h1>
+  </section>
+  <section class="intro">
+    <p id="intro-heading">
+      Hello! I am a software developer! I can help you build a product, feature or website. Taka a look of my works.
+      If you like what you see and have a project you need coded, don’t hesitate and contact me.
+    </p>
+  </section>
+  <div class="button">
+    <a id="btn" href="">Start collaboraton</a>
+  </div>
```

Comment:
You are almost there @Kashif-Saleem-Ghuman. Let's make the whole of this to be one section. Your name, the paragraph, and the button should all be part of one section. We often call this first section the `hero section`. It is the landing page of your portfolio.

Please refactor your code as below and we'll be good to go:

```JavaScript
<section class="heading">
  <h1 id="name">Kashif Saleem</h1>
  <p id="intro-heading">
    Hello! I am a software developer! I can help you build a product, feature, or website. Taka a look at my work.
    Don't hesitate to contact me if you like what you see and have a project you need to be coded.
  </p>
  
  <div class="button">
    <a id="btn" href="">Start collaboraton</a>
  </div>
</section>
```